src/main/kotlin/io/sledge/deployer/commands/SledgeCommand.kt

```diff
@@ -6,25 +6,30 @@ import com.github.ajalt.clikt.parameters.options.option
 import io.sledge.deployer.core.api.Deployer
 import io.sledge.deployer.core.api.Deployment
 import io.sledge.deployer.core.api.SledgeFileParser
+import io.sledge.deployer.crx.CrxConfiguration
+import io.sledge.deployer.crx.CrxDeployer
 import java.io.File
 
-class SledgeCommand(val sledgeFileParser: SledgeFileParser, val deployer: Deployer) : CliktCommand(name = "sledge") {
+class SledgeCommand(val sledgeFileParser: SledgeFileParser, val deployer: CrxDeployer) : CliktCommand(name = "sledge") {
     private val deploymentDefinitionName by argument(name = "DEPLOYMENT_DEFINITION_NAME", help = "The name of the Deployment definition, e.g. local, dev-auhor, dev-publish, test, prod, etc.")
     private val targetServer by argument(name = "TARGET_SERVER", help = "The url to the target server, e.g. http://server:4502")
     val user by option()
     val password by option()
+    val deploymentMode by option()
```

Comment:
deploymentMode is not really used in the code or?
What would be the idea?

Reply:
yes, not yet. I was asking Tinu if there is a need for a particular action like only install or only uninstall but no so far. We could remove it then for MVP version